layouts/partials/openapi/render-operation.html

```diff
@@ -59,7 +59,22 @@ <h1 id="{{ lower $method }}{{ $anchor }}">
   </tr>
  <tr>
   <th>Requires authentication:</th>
-  <td>{{ if $operation_data.security }}Yes{{ else }}No{{ end }}</td>
+  {{/*
```

Comment:
Since you're rewriting the logic here, how much of a pain would it be to make it say "Requires authentication: optional" if it's optional?

Reply:
It wouldn't be complicated, but isn't this wording weird?

Reply:
"No" feels like there's *no* option for authentication, even if technically correct. "Optional" or "conditional" would be more suitable.

Reply:
Done